src/middlewared/middlewared/plugins/virt/instance.py

```diff
@@ -580,9 +580,14 @@ async def start(self, job, id):
     @private
     async def start_impl(self, job, id):
         instance = await self.middleware.call('virt.instance.get_instance', id)
+        if instance['status'] not in ('RUNNING', 'STOPPED'):
```

Comment:
@Qubad786 why are we checking for `RUNNING` here? This is the `start_impl` method so it makes me think we should only be checking for `STOPPED`?

Reply:
@anodos325 fair point, we should only do this when we are in `STOPPED` state?

Reply:
I believe you added the check for RUNNING in b1700abd0b73ff023550061d02efaae303a6a98b

Reply:
@anodos325 the `RUNNING` check added was for to apply idmaps and this specific case is for raising a validation here, that check also only applies idmap settings if the instance is not in running state and this specific check is different here at L583?